modules/ding_webtrekk/ding_webtrekk.js

```diff
@@ -154,11 +159,11 @@
         });
 
         $('.slick-arrow', this).once('js-ding-webtrekk').click(function() {
-          var linkId = 'Karousel, click på forrige knappen';
+          var linkId = Drupal.t('Carousel, clicked on previous link');
           var wtkId = DING_WEBTREKK_PARAMETER_CAROUSEL_PREV;
 
           if ($(this).hasClass('slick-next')) {
-            linkId = 'Karousel, click på næste knappen';
```

Comment:
As I understand it these are strings used by administrators in Webtrekk - not end users on the site. Is there a reason why they should pass through the translation engine?

If someone changes the translation without knowing where it is sed I fear will cause unwanted changes in Webtrekk reporting.

Reply:
K, it's just wrong to use danish inside JS code.

Reply:
But, reverted back.